amethyst_rendy/src/formats/texture.rs

```diff
@@ -76,6 +76,7 @@ impl Default for ImageFormat {
                 },
                 comparison: None,
                 border: PackedColor(0),
+                normalized: false,
```

Comment:
This is a mistake, the former behaviour is supposed to be true and otherwise it's unsupported in some backends.

https://github.com/gfx-rs/gfx/blob/4a91f35e3df9b00e24d5cee65e5911977e336b0e/src/backend/dx11/src/device.rs#L1942